src/registrar/views/domain_request.py

```diff
@@ -379,6 +379,11 @@ def get_context_data(self):
         if self.domain_request.requested_domain is not None:
             requested_domain_name = self.domain_request.requested_domain.name
 
+        text_domain_name = str(requested_domain_name) if requested_domain_name is not None else None
+
+        if text_domain_name and len(text_domain_name) > 30:
+            text_domain_name = text_domain_name[:30] + "\n" + text_domain_name[30:]
+
         context_stuff = {}
         if DomainRequest._form_complete(self.domain_request, self.request):
             modal_button = '<button type="submit" ' 'class="usa-button" ' ">Submit request</button>"
```

Comment:
Let me know if you already tried this and it didn't work, but I would suggest handling overflow text by adding a class to the modal button that has a maxlength and handles overflow text since the 30 char limit could be variable if we ever change our modal length! You can reuse or reference Rebeca's domain-name-wrap class she made in [a similar PR](https://github.com/cisagov/manage.get.gov/pull/2825/files)!

Reply:
Ahh thanks for pointing that out! I tried to do it with using max-width:100%, and the break word css styling but it didn't work. Rebecca's wrap class worked for me!